Update draft-brw-sconepro-rate-policy-discovery.md (#28)
boucadair/draft-xxx-ac-rate-policy-discovery: draft-brw-sconepro-rate-policy-discovery.md

> ISPs are prohibited from blocking or slowing down of Internet traffic, except where necessary. The exceptions are limited to: traffic management to comply with a legal order, to ensure network integrity and security, and to manage congestion, provided that equivalent categories of traffic are treated equally.

Some additional operational considerations are detailed in the following sub-sections.

## Architectural aspects

> Approaches based on middleboxes are not recommended due to its inherent limitations, in terms of performance, scalability, redundancy, etc. Moreover, the management and operation of such middleboxes remain unclear, which can motivate operational issues and responsibilities.

Reviewer: ```suggestion
Approaches based on middleboxes are generally not recommended due to their inherent limitations, in terms of performance, scalability, redundancy, etc. Moreover, the management and operation of such middleboxes remain unclear, which can motivate operational issues and responsibilities.
```

Finally, signals could not be required at every moment, but only in situations that can benefit the service. Such situations could be due, for instance, to given levels of congestion, or based on previous information shared by the application (e.g., SLO thresholds) so that signals can be triggered according to service conditions.

## Signal estimation

The validity of the estimation produced by the network could be questioned by the application. Trust is required in a way that applications can safely follow guidance from the network. Furthermore, whatever estimation should be timely produced, avoiding the generation of aged estimations that could not correspond to the actual service circumstances. Finally, some common guidance is necessary to define an standard way of generating signals, for instance, per-flow or per group of flows.

An open point is how to deal with adaptive applications, in the sense that signals could not be of value because the self-adaptation nature of the application.

> ## Signal “interference”

Reviewer: ```suggestion
## Signal “Interference”
```

The network is built on multiple layers. In some cases, different solutions targeting similar objectives (e.g., congestion control or bottleneck mitigation) can be in place. It is then necessary to assess the simultaneous coexistence of these solutions to avoid contradictory effects or “interferences”.

> ## Privacy and security aspects

Reviewer: ```suggestion
```

> Network signals represent sensitive information subject of malicious behavior or creating vulnerabilities. The signals offered should be protected to avoid any of these concerns, and the consumers of these signals should be authenticated. 

Reviewer: ```suggestion
```
